docroot/themes/humsci/humsci_basic/src/scss/_base.scss

```diff
@@ -1,8 +1,18 @@
 @charset "UTF-8";
 
-// Decanter's use of global variables won't allow us to import with @use
-// @use 'decanter/core/src/scss/components/brand-bar/index';
-@import 'decanter/core/src/scss/decanter-no-markup';
```

Comment:
we can bring in these as well, I think since every site needs to have them:

- brand bar https://github.com/SU-SWS/decanter/tree/v6/core/src/scss/components/brand-bar
- lockup https://github.com/SU-SWS/decanter/tree/v6/core/src/scss/components/lockup
- footer things https://github.com/SU-SWS/decanter/tree/v6/core/src/scss/components/global-footer and https://github.com/SU-SWS/decanter/tree/v6/core/src/scss/components/local-footer

Reply:
we might want to consider using the navigation too https://github.com/SU-SWS/decanter/tree/6.0.0/core/src/scss/components/secondary-nav https://github.com/SU-SWS/decanter/tree/6.0.0/core/src/scss/components/main-nav

Reply:
this is accompanied by some JS we can import too if needed

Reply:
I'd like feedback on what @nrambeck or @bryanbraun thinks about bringing these in